pkg/api/soc.go

```diff
@@ -140,8 +141,8 @@ func (s *Service) socUploadHandler(w http.ResponseWriter, r *http.Request) {
 		s.logger.Debug("soc upload: get postage batch issuer failed", "batch_id", hex.EncodeToString(batch), "error", err)
 		s.logger.Error(nil, "soc upload: get postage batch issue")
 		switch {
-		case errors.Is(err, postage.ErrNotFound):
-			jsonhttp.BadRequest(w, "batch not found")
+		case errors.Is(err, storage.ErrNotFound):
+			jsonhttp.NotFound(w, "batch not found")
```

Comment:
usually in default statement we us `jsonhttp.InternalServerError`, but in this case we don't. is this intentionally?